### DISCUSSION TITLE
PromptClient: Support mustache type export to langchain

### DISCUSSION THREAD

**Discussion by maxjacu** [langfuse/langfuse#1774] [Ideas]
### Describe the feature or potential improvement

Langchain as of recent supports (finally) mustache style variables {{var}}. Langfuse would have to implement some minor changes to export ChatPrompts correctly into that format.

**ChatPromptClient**
Was able to simplify conversion of get_langchain_prompt to `[(msg["role"], msg["content"]) for msg in prompt]`. 

**TextPromptClient**
Could just return the prompt, so no need for the helper function if mustache is used.

**Comment by marcklingen** [comment]
Turning this into an issue, thanks for sharing!
